### DISCUSSION TITLE
Is there a way I can plot the Inlet temperature?

### DISCUSSION THREAD

**Discussion by smaca69** [artisan-roaster-scope/artisan#1249] [General]
Hi,

Is there a way I can plot the Inlet temperature on the screen even though it is 200c  higher than the BT & ET temps?

Cheers

**Comment by roasterdave** [comment]
You can increase the temperature axis maximum to show it (Config>> Axes).  The downside is less visual resolution for the other curves.

Another option is to use a symbolic equation, like `x-200` to subtract 200 from the inlet temperature.  This allows the inlet temperature cure to show without changing the y-axis.    Of course you'll have to remember that the actual inlet temperature is 200 degrees higher than what the graph shows  Add the equation in the y(x) box for the extra device the reads your inlet temperature. 

More information on symbolic equations can be found here in the [Artisan Quick Start Guide](https://artisan-scope.org/docs/setup/).

**Reply by smaca69** [reply]
ok, thank you.  What is the variable called for the Inlet temp?

**Reply by roasterdave** [reply]
You have provided no information about your setup so it's not possible to answer that question.  However, assuming the inlet temperature is setup up as an extra device (Config>> Device>> Extra Devices tab) you can put the equation above into the y(x) field for the inlet temperature and it will be scaled accordingly when you roast.  Probably best to change the text Label for inlet temperature to indicate it is IT minus 200.
